index.html

```diff
@@ -0,0 +1,222 @@
+        <div>
+            <h1>Smashing <br>
+                Magazine</h1>
+        </div>
+        <nav>
+            <ul>
+                <li>
+                    <h2> Articles</h2>
+                </li>
+                <li>
+                    <h2>BOOKS</h2>
+                </li>
+                <li>
+                    <h2>EVENTS</h2>
+                </li>
+                <li>
+                    <h2>JOBS</h2>
+                </li>
+                <li>
+                    <h2>ME</h2>
+                </li>
+            </ul>
+        </nav>
+        <div class="topics">
+            <h3>TOPICS</h3>
+        </div>
+
+
+    </header>
+
+    <main>
+        <h1 class="miss">
+            <span class=" w7">Don’t Miss These
+                Latest</span><br>
+            <span class=" w7">Article</span>
+        </h1>
+
+        <section class="articles">
+            <h2 class="miss">
+                <span class=" w7">Don’t Miss These
+                    Latest</span><br>
+                <span class=" w7">Article</span>
+            </h2>
+            <section class=" w3">
+                <h2>Monthly Web Development Update 6/2019: Rethinking Privacy And User Engagement</h2>
+            </section>
+            <section class=" w4">
+                <h2>Inspired Design Decisions: Avaunt Magazine</h2>
+            </section>
+            <section class=" w5">
+                <h2>Bringing A Healthy Code Review Mindset To Your Team</h2>
+            </section>
+            <section class=" w3">
+                <h2>UX Optimizations For Keyboard-Only And Assistive Technology Users</h2>
+            </section>
+        </section>
+        <div class="latest-post">
+            <section class="w1">
+                <h2>Styling In Modern Web Apps
+                </h2><br><br>
+                Lorem ipsum dolor sit amet consectetur adipisicing elit. Fugiat nostrum sapiente odio, reiciendis, culpa
+                numquam tempora omnis cumque recusandae repudiandae architecto. Similique reiciendis eos, ipsum
+                accusamus atque odio at molestias!
+            </section>
+            <section class="w1">
+                <h2>Web Accessibility In Context</h2><br><br>
+                Lorem ipsum dolor sit amet consectetur adipisicing elit. Fugiat nostrum sapiente odio, reiciendis, culpa
+                numquam tempora omnis cumque recusandae repudiandae architecto. Similique reiciendis eos, ipsum
+                accusamus atque odio at molestias!
+            </section>
+            <section class="w1">
+                <h2>How To Start Using Sketch And Framer X</h2><br><br>
+                Lorem ipsum dolor sit amet consectetur adipisicing elit. Fugiat nostrum sapiente odio, reiciendis, culpa
+                numquam tempora omnis cumque recusandae repudiandae architecto. Similique reiciendis eos, ipsum
+                accusamus atque odio at molestias!
+            </section>
+
+            <section class="w3">
```

Comment:
This latest post section must be similar to the original page, meaning that should have only 3 articles with the same width (33% of the page) each. I notice that you used 3 `section` tags here. There is no need for it since it's just one section. Make the `<div class="latest-post">` a section tag and divide the 3 articles using the `article` tag. Delete this last item since this section should contain only 3 articles with the same width.